processing/src/main/java/org/apache/carbondata/processing/newflow/iterators/CarbonArrayWritable.java

```diff
@@ -0,0 +1,51 @@
+package org.apache.carbondata.processing.newflow.iterators;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.Arrays;
+
+import org.apache.hadoop.io.Writable;
+
+/**
+ * It is hadoop's writable value wrapper.
+ */
+public class CarbonArrayWritable implements Writable {
```

Comment:
Why this is carbon-processing but not carbon-hadoop module? All hadoop related class should be in carbon-hadoop module